src/Elcodi/Common/VisithorBridgeBundle/Visithor/EnvironmentBuilder.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014-2015 Elcodi.com
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Common\VisithorBridgeBundle\Visithor;
+
+use Doctrine\DBAL\Connection;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Visithor\Bundle\Environment\Interfaces\EnvironmentBuilderInterface;
+
+/**
+ * Class EnvironmentBuilder
+ */
+class EnvironmentBuilder implements EnvironmentBuilderInterface
+{
+    /**
+     * @var boolean
+     *
+     * Debug mode
+     */
+    protected $debug = false;
+
+    /**
+     * Set up environment
+     *
+     * @param KernelInterface $kernel Kernel
+     *
+     * @return $this Self object
+     */
+    public function setUp(KernelInterface $kernel)
+    {
+        gc_collect_cycles();
+
+        $application = new Application($kernel);
```

Comment:
you are creating another application instance, if you are running this as a command there is already one, all those commands can be found and executed, no need for doing it like this ...